src/store/actionCreators/usersActionCreators.js

```diff
@@ -0,0 +1,35 @@
+import { getAllUsers, removeUserData, editUser, createUser } from '../../services/users-services ';
+import { EDIT_USER, GET_USERS, REMOVE_USER } from '../actions/actions';
+
+const getUsers = (users) => ({ type: GET_USERS, payload: { users } });
+const removeUser = (userId) => ({ type: REMOVE_USER, payload: { userId } });
+const editUserData = (userId, userData) => ({ type: EDIT_USER, payload: { userId, userData } });
+
+export function getUsersThunk() {
+  return async (dispatch) => {
+    const users = await getAllUsers();
```

Comment:
where do you handler possible errors ? I don't see it.

Reply:
Error handling in getAllUsers(). Should I process them here too?

Reply:
if it's handled there than you can omit it here